circle.yml

```diff
@@ -0,0 +1,13 @@
+machine:
+  node:
+    version: 6.9
+  environment:
+    PATH: "${PATH}:${HOME}/${CIRCLE_PROJECT_REPONAME}/node_modules/.bin"
+dependencies:
+  override:
+    - yarn
+  cache_directories:
+    - ~/.cache/yarn
```

Comment:
Funny - I have both this and `~/.yarn-cache` on my local machine.  Did they switch recently?